test/test_jit_fuser_te.py

```diff
@@ -1936,6 +1960,22 @@ def bn_neither(i, x):
             for fn in [bn, bn_no_weight, bn_no_bias, bn_neither]:
                 test(fn, (i, x))
 
+    @unittest.skipIf(not RUN_CUDA, "fuser requires CUDA for fp16")
+    def test_bn_half_61382(self):
```

Comment:
We already have an equivalent test, it's probably better to just re-enable it:
https://github.com/pytorch/pytorch/blob/44e3ed88c9a1bd9ee6b0168ba5271a2c6b006cc8/test/test_tensorexpr.py#L1225-L1241